README.md

```diff
@@ -1,8 +1,8 @@
-# Keda Manager
+# Keda module
 
 ## Overview 
 
-Keda Manager is an extension to the Kyma ecosystem that allows users to install KEDA. It follows the Kubernetes operator pattern to manage the lifecycle of the KEDA installation based on the existence and the content of the dedicated Keda custom resource (CR).
+Keda module consists of Keda Manager that is an extension to the Kyma runtime. It allows users to install KEDA. It follows the Kubernetes operator pattern to manage the lifecycle of the KEDA installation based on the existence and the content of the dedicated Keda custom resource (CR).
 
 ![a](./docs/assets/keda-overview.drawio.svg)
```

Comment:
The diagram is not formatted according to [our guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/09-diagrams.md).